### ISSUE TITLE
Test Py2.7 gcc493 architecture on DQMGUI

### ISSUE THREAD

**Issue by rovere** [rovere/dqmgui#28]

**Comment by rovere** [comment]
I've successfully built 8.2.4-rc.1 RPMs using the new architecture following instructions [here](https://cms-http-group.web.cern.ch/cms-http-group/tutorials/environ/dmwm-build.html).

The RPMs have been uploaded [[here]](http://cmsrep.cern.ch/cmssw/comp.rovere/RPMS/slc6_amd64_gcc493/).

It would be nice to install and test it.

**Comment by rovere** [comment]
@vanbesien 
feel free to comment to this ticket your findings about running with py2.7

**Comment by vanbesien** [comment]
Tested last week and was unable to start the server.
There was only one issue and after a bypass/hack all the rest seemed to work.

I'll copy paste my info from the chat below:
ok, I pinpointed the exact location of the problem: line 532 in HG1509j/sw.rovere/slc6_amd64_gcc493/external/python/2.7.6-comp/lib/python2.7/inspect.py
might be a few lines higher in the original code, since I added some print statements
apparently, the GUI, when initializing, inspects all the loaded modules
for the modules that start with monitoring, it actually tries to read the lines
if it's binary, there is nothing to read
the old code didn't care that much when the sourcecode could not be read, the code is:
file = getsourcefile(object) or getfile(object)
where getsourcefile() returns none and getfile() returns the filename
the new code does exactly the same thing, but inbetween raises an IOError when getsourcefile() returns None
which is annoying
basically saying, well, if we can't get the code, we just continue with the file as is...
but there is a nuance: it only raises the error when  Getsourcefile() returns none AND the file is not surrounded by "<>"
I'm sure we'll be able to crack it somewhere around site-packages/Monitoring/Core/GUI.py:270
